doc/releases/release-notes-4.0.rst

```diff
@@ -168,17 +187,50 @@ Boards & SoC Support
   * Added STM32U0 series with GPIO, Serial, I2C, DAC, ADC, flash, PWM and counter driver support.
   * Added STM32WB0 series with GPIO, Serial, I2C, SPI, ADC, DMA and flash driver support.
   * Added STM32U545xx SoC variant.
+  * NXP i.MX93's Cortex-M33 core
 
 * Made these changes in other SoC series:
 
   * NXP S32Z270: Added support for the new silicon cut version 2.0. Note that the previous
+  * NXP s32k3: fixed RAM retention issue
+  * NXP s32k1: obtain system clock frequency from Devicetree
     versions (1.0 and 1.1) are no longer supported.
   * Added ESP32 WROVER-E-N16R4 variant.
   * STM32H5: Added support for OpenOCD through STMicroelectronics OpenOCD fork.
   * MAX32: Enabled Segger RTT and SystemView support.
   * Silabs Series 2: Use oscillator, clock and DCDC configuration from device tree during init.
   * Silabs Series 2: Added initialization for SMU (Security Management Unit).
   * Silabs Series 2: Use sleeptimer as the default OS timer instead of systick.
+  * NXP i.MX8MP: Enable the IRQ_STEER interrupt controller.
+  * NXP RWxxx:
+      * added additional support to Wakeup from low power modes
+      * RW61x: increased main stack size to avoid stack overflows when running BLE
+      * RW612: enabled SCTIMER
+  * NXP IMXRT:
+
+    * Fixed flexspi xip configuration issue regarding code relocation due to order
+    * Fixed flexspi boot issue caused by am erroneous relocation of the Flash Configuration Block
+      of Kconfig defaults being sourced
+
+  * NXP RT11xx:
+
+    * fixed clock speeds for rt117x and rt116x.
+    * enabled FlexIO
+
+  * NXP MIMXRT1010:
+
+      * Fixed the eDMA channel definition (dts) to include a specific interrupt
+      * Added default offset to fix the platform not booting because RT1011 expects its flash configuration
+        block to be at a different offset than the rest of the RT10xx series.
+
+  * NXP IMXRT116x: Fixed bus clocking to align with the settings of the MCUXpresso SDK
+  * NXP mimxrt685: fixed clocks to enable DMIC
+  * NXP MCX C Series:
+
+    * Replaced PLATFORM_SPECIFIC_INIT with PLATFORM_RESET_HOOK
```

Comment:
wasn't this done for every platform in the tree?

Reply:
im pretty sure that not only was this done for every platform in tree but also the entire  MCX C was added during this development cycle 

#78235
#76865